tests/unit_tests/test_tree.py

```diff
@@ -66,11 +66,11 @@ def test_parse_tree():
         "from": 0,
         "query": {
             "bool": {
-                "filter": [{"term": {"path": node_id_biology}}],
-                "must": [
-                    {"match": {"permissions.Read": "GROUP_EVERYONE"}},
-                    {"match": {"properties.cm:edu_metadataset": "mds_oeh"}},
-                    {"match": {"nodeRef.storeRef.protocol": "workspace"}},
+                "filter": [
```

Comment:
Elastic Query language is still like Klingon to me.... Does the new `base_filter` change anything on the outcome or query performance? Or is it just a more flexible way to put together this way?

Reply:
It does - at least in theory.
"Filter" is like "must" but without calculation of score, i.e., the accuracy of the found search result. Therefore the elasticsearch has less to do.
"Match" is like "term", but match includes a fuzzy search, so term is faster if we are certain, that the keyword is exactly like we want it.